wp-content/themes/rebalance-child-mmn/functions.php

```diff
@@ -246,6 +246,80 @@ function suppress_feature_image(){
   }
 }
 
+/*mmn: merites blog conditional formatting*/
+function add_custom_font() {
+	//mmn: add custom font for merites
+    $font_url = get_stylesheet_directory_uri() . '/merites/WindsorConDExtBol.otf';
+    $font_family = 'WindsorConDExtBol';
+
+    $font_formats = array(
+        'otf' => 'opentype',
+    );
+
+    $font_style = '@font-face {
+        font-family: "' . $font_family . '";
+        src: url("' . esc_url($font_url) . '") format("' . implode(', ', $font_formats) . '");
+        font-weight: normal;
+        font-style: normal;
+    }';
+
+    wp_add_inline_style('rebalance-style', $font_style);
+}
+add_action('wp_enqueue_scripts', 'add_custom_font');
+
+function is_merites() {
+	//mmn: helper function for deciding on rendering the normal site or merites subblog
+	//returns true if standing on an article written by the user 'merites'
+	//or if standing on the author page (/author/merites)
+	$merites_path = get_stylesheet_directory_uri() . '/merites';
+	$author_id = get_post_field('post_author', get_the_ID());
+	if (is_author('merites') || ( is_single() && $author_id == 69 )) {
```

Comment:
I'm not sure I fully understand so help me unpack this
1) Why would the `author_id` be different in some local setups? The author Merites is already created in our db so it should never be the case that anybody else gets the id 69 and testing this feature wouldn't make sense unless you use a db that already contains this author. I haven't found a way for using the username (merites) for this condition and all the other features (nicename = Merítés) can change anytime so I think the id is the safest to use. 
2) The `is_author()` function should take care of the `/author/merites` view and the `is_single() && $author_id == 69` is responsible for displaying a single article. It's not clear to me whether the problem on your side is due to a differing `author_id` on your local setup or something else.

Reply:
These are good points but I there is still a valid "design for testibility" issue here. I'm not working with a fresh DB import and it would be nice if having one would not be a requirement for the local setup. My 69th user does exist :)

Can't we use the user name of the display name? Is there a function to check them for the author? I think using them would be equally stable, more robust across local installations, and a more readable code too.